ci/goreleaser/goreleaser.yml

```diff
@@ -16,6 +16,18 @@ builds:
     goarch:
       - amd64
     binary: tyk
+  - id: fips-linux
```

Comment:
@alephnull @asutosh any advices on how to fix this issue?
https://github.com/TykTechnologies/tyk/actions/runs/4822243408/jobs/8589212246?pr=4778

It fails with:
```
⨯ release failed after 5m36s               error=nfpm failed for tyk-gateway_0.0.0~SNAPSHOT~7346ab7_amd64.deb: add globbed files from "/__w/tyk/tyk/dist/fips-linux_linux_amd64_v1/tyk": adding  at destination /opt/tyk-gateway/tyk: file with source /__w/tyk/tyk/dist/std-linux_linux_amd64_v1/tyk is already present at this destination: content collision
```

I can assume that because it is same goos and goarch it looks same to goreleaser?

Reply:
You're going to have to build a different binary, see how the PAYG version is built in tyk-analytics for example.